### DISCUSSION TITLE
[Question] Is there a way to run inGameWindow as npm run start?

### DISCUSSION THREAD

**Discussion by pekac45** [AlbericoD/overwolf-modern-react-boilerplate#30] [Q&A] [answered]
I am trying to develop the inGameWindow but when I run npm run start, the http://localhost:3000/ is only showing desktop app icon. How can I see the inGameWindow in the browser/overlay, so I can use hot reload features?

**Comment by San-Moshe** [comment, marked as answer]
Hey,
I used dev environment to manually set the current page to InGameWindow.

In src/app/app.tsx add the following line inside the if block:
 
 `setPage​(​WINDOW_NAMES​.​INGAME​)`

**Reply by pekac45** [reply]
Nice one, thank you for pointer. 

I have ended up doing

```
if (process.env.NODE_ENV === "development") {
  if (process.env.REACT_APP_INGAME === "true") {
    setPage(WINDOW_NAMES.INGAME);
  } else {
    //you can set the current window to dev if you want to see the dev page <Normal Browser>
    setPage(WINDOW_NAMES.DESKTOP);
  }
}
```

and adding cross-env flag `REACT_APP_INGAME=true` in a package.json script.
